person.rb

```diff
@@ -22,6 +23,11 @@ def correct_name
     name
   end
 
+  def add_rental(rental)
+    @rentals.push(rental)
+    rental.person = self
+  end
+
```

Comment:
- [  ] Please Getters and Setters for @rentals should be present you can add the rentals to attr_accessor also